File: uri/media_tres/media_tres.java
import java.io.IOException;
import java.util.*;

public class Main{
    
    private static double pesoA = 2.0;
    private static double pesoB = 3.0;
    private static double pesoC = 5.0;
    
    private static double media( double notaA, double notaB, double notaC )
    {
        double numMedia;
        numMedia = ((notaA * pesoA) + (notaB * pesoB) + (notaC * pesoC)) / (pesoA + pesoB + pesoC);
        return numMedia;
    }
    
    public static void main(String[] args) throws IOException {     
        Locale.setDefault(Locale.US);
        Scanner inpoot = new Scanner(System.in);
        double a, b, c, nmedia;
        a = inpoot.nextDouble();
        b = inpoot.nextDouble();
        c = inpoot.nextDouble();
        nmedia = media(a, b, c);
        System.out.printf("MEDIA = %.1f\n", nmedia);
    }    
}

File: uri/media_tres/media_tres.cpp

#include <iostream>
#include <iomanip>

using namespace std;

class Namedia{

    private:

        double pesoA;
        double pesoB;
        double pesoC;

        double calculamedia( double notaA, double notaB, double notaC )
        {
            double poo;
            poo =  ((notaA * pesoA) + (notaB * pesoB) + (notaC * pesoC) )/ (pesoA + pesoB + pesoC);
            return poo;
        }

    public:

        Namedia(){
            pesoA = 2.0;
            pesoB = 3.0;
            pesoC = 5.0;
        }

        void exibir()
        {
            double notaA, notaB, notaC, media;
            cin >> notaA;
            cin >> notaB;
            cin >> notaC;
            media = calculamedia(notaA, notaB, notaC);
            cout << "MEDIA = "<< std::fixed << setprecision(1) << media << "\n";
        }
};

int main(void)
{
    Namedia foo;
    foo.exibir();
    return 0;
}






